ui/src/literature/components/__tests__/ImprintInfo.test.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
```

Comment:
Can you add a test when you'll have couple of imprints info in the array?

Reply:
updated